src/components/app/routes/queries/canRedeemCourse.js

```diff
@@ -1,46 +1,21 @@
-import { camelCaseObject, getConfig } from '@edx/frontend-platform';
-import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
-
-import { getErrorResponseStatusCode } from '../../../../utils/common';
+/* eslint-disable no-underscore-dangle */
```

Comment:
nit: I would recommend disabling this ESLint error globally within the `.eslintrc.js` file instead of disabling the rule altogether in all the relevant files, specifically for the `.ctx` property.

```js
{
  rules: {
    // ...
    'no-underscore-dangle': ['error', { allow: ['_ctx'] }],
  }
}
```